src/stories/containers/cu-about/cu-about-container.tsx

```diff
@@ -63,14 +65,30 @@ const CuAboutContainer = ({ code, cuAbout, contributors }: Props) => {
 
   return (
     <ContainerAbout isLight={isLight}>
-    <Head>
-      <title>About Sustainable Ecosystem Scaling Core Unit at MakerDAO</title>
-      <link rel="icon" href="/favicon.png" />
-      <meta property='og:site_name' content="About Sustainable Ecosystem Scaling Core Unit at MakerDAO"/>
-      <meta name="description" content="Learn about the Sustainable Ecosystem Scaling Core Unit at MakerDAO: their mandate, vision, mission, strategy, and more." />
-      <meta name="og:description" content="Learn about the Sustainable Ecosystem Scaling Core Unit at MakerDAO: their mandate, vision, mission, strategy, and more." />
-      <meta name="robots" content="index,follow"/>
-    </Head>
+      <Head>
+        <title>{`About ${cuAbout.name} Core Unit at MakerDAO`}</title>
+        <link rel="icon" href={cuAbout.image} />
+        <meta
+          property="og:site_name"
```

Comment:
Maybe you should use `og:title` instead. `og:site_name` is used for the site name (eg: MakerDAO SES)

https://ogp.me/

Reply:
you are right , i will change it